tests/testthat/test-enw_model.R

```diff
@@ -1,14 +1,19 @@
 test_that("enw_model can compile the default model", {
   skip_on_cran()
-  enw_model()
+  expect_error(enw_model(), NA)
```

Comment:
do we?

Reply:
This is the hacky way to test it doesn't error.